lib/instrumentation/transaction.js

```diff
@@ -82,7 +82,7 @@ Transaction.prototype.setCustomContext = function (context) {
   this._custom = Object.assign(this._custom || {}, context)
 }
 
-Transaction.prototype.startSpan = function (name, type, opts) {
+Transaction.prototype.startSpan = function (name, ...args) {
```

Comment:
Remember to update the TypeScript typings as well

Reply:
fixed